File: app/src/main/java/com/waveneuro/ui/dashboard/help/HelpActivity.java
package com.waveneuro.ui.dashboard.help;

import android.os.Bundle;

import androidx.annotation.Nullable;

import com.waveneuro.R;
import com.waveneuro.ui.base.BaseActivity;
import com.waveneuro.ui.dashboard.more.WebCommand;

import javax.inject.Inject;

import butterknife.ButterKnife;
import butterknife.OnClick;

public class HelpActivity extends BaseActivity {

    @Inject
    WebCommand webCommand;

    @OnClick(R.id.llFaq)
    public void onClickFaq() {
        webCommand.navigate(WebCommand.PAGE_FAQ);
    }

    @OnClick(R.id.llPrivacyPolicy)
    public void onClickPrivacyPolicy() {
        webCommand.navigate(WebCommand.PAGE_POLICY);
    }

    @OnClick(R.id.llTerms)
    public void onClickTerms() {
        webCommand.navigate(WebCommand.PAGE_TERMS_CONDITIONS);
    }

    @OnClick(R.id.btnContact)
    public void onClickContact() {
        webCommand.navigate(WebCommand.PAGE_CONTACT);
    }

    @OnClick(R.id.iv_back)
    public void OnClickBack() {
        onBackPressed();
    }

    @Override
    protected void onCreate(@Nullable @org.jetbrains.annotations.Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        activityComponent().inject(this);
        setContentView(R.layout.activity_help);
        ButterKnife.bind(this);
    }
}

File: app/src/main/java/com/waveneuro/ui/dashboard/help/HelpActivity.kt
package com.waveneuro.ui.dashboard.help

import android.content.Context
import android.content.Intent
import android.os.Bundle
import com.waveneuro.databinding.ActivityHelpBinding
import com.waveneuro.ui.base.activity.BaseActivity
import com.waveneuro.ui.dashboard.web.WebActivity
import com.waveneuro.ui.dashboard.web.WebActivity.Companion.PAGE_CONTACT
import com.waveneuro.ui.dashboard.web.WebActivity.Companion.PAGE_FAQ
import com.waveneuro.ui.dashboard.web.WebActivity.Companion.PAGE_POLICY
import com.waveneuro.ui.dashboard.web.WebActivity.Companion.PAGE_TERMS_CONDITIONS

class HelpActivity : BaseActivity<ActivityHelpBinding>() {

    override fun initBinding(): ActivityHelpBinding = ActivityHelpBinding.inflate(layoutInflater)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setView()
    }

    private fun setView() {
        with(binding) {
            llFaq.setOnClickListener {
                startActivity(WebActivity.newIntent(this@HelpActivity, PAGE_FAQ))
            }
            llPrivacyPolicy.setOnClickListener {
                startActivity(WebActivity.newIntent(this@HelpActivity, PAGE_POLICY))
            }
            llTerms.setOnClickListener {
                startActivity(WebActivity.newIntent(this@HelpActivity, PAGE_TERMS_CONDITIONS))
            }
            btnContact.setOnClickListener {
                startActivity(WebActivity.newIntent(this@HelpActivity, PAGE_CONTACT))
            }
            ivBack.setOnClickListener {
                onBackPressed()
            }
        }
    }

    companion object {
        fun newIntent(context: Context) = Intent(context, HelpActivity::class.java)
    }

}